charts/influxdb2/values.yaml

```diff
@@ -1,5 +1,6 @@
 image:
-  repository: quay.io/influxdb/influxdb
+  repository: influxdb
+  # tag:
```

Comment:
Did you consider using `2.0.4-alpine` as a default?

Reply:
I will leave this up to the maintainers.

Reply:
yeah...there are two options here `2.0-alpine` which means the maintenance revisions will be automatically pulled.
or `2.0.4-alpine` which means for every maintenance release, we'll need to update the chart.

But, we need this to be tagged...latest is undesirable.

Reply:
As it is it will default to the charts app version. I will change it to only be configured with the tag value and use 2.0.4-alpine